### ISSUE TITLE
Not a bootable disk

### ISSUE THREAD

**Issue by amiyadav** [cfenollosa/os-tutorial#219]
I am running --> qemu-system-x86_64 -drive format=raw,file=boot_sect_hello.bin

But it is showing me

<img width="717" alt="Screenshot 2021-09-07 at 12 02 37 AM" src="https://user-images.githubusercontent.com/6071737/132252873-a46de35e-f689-4b95-88e2-74121d9fd725.png">

**Comment by BetaPictoris** [comment]
Have you tried `qemu boot_sect_hello.bin`?

**Comment by amiyadav** [comment]
yeah alone qemu is not working instead I have to use  `qemu-system-x86_64 boot_sect_hello.bin` which gives me Not a bootable disk

**Comment by BetaPictoris** [comment]
> yeah alone qemu is not working instead I have to use `qemu-system-x86_64 boot_sect_hello.bin` which gives me Not a bootable disk.
> 
> I am doing this on MAC

What command are you using to build the `boot_sect_hello.bin` file?

**Comment by amiyadav** [comment]
> > yeah alone qemu is not working instead I have to use `qemu-system-x86_64 boot_sect_hello.bin` which gives me Not a bootable disk.
> > I am doing this on MAC
> 
> What command are you using to build the `boot_sect_hello.bin` file?

nasm -fbin boot_sect_hello.asm -o boot_sect_hello.bin

**Comment by BetaPictoris** [comment]
> > > yeah alone qemu is not working instead I have to use `qemu-system-x86_64 boot_sect_hello.bin` which gives me Not a bootable disk.
> > > I am doing this on MAC
> > 
> > 
> > What command are you using to build the `boot_sect_hello.bin` file?
> 
> nasm -fbin boot_sect_hello.asm -o boot_sect_hello.bin

What was the output, if any?

**Comment by swap2ag** [comment]
@amiyadav Can you place the file `boot_sect_hello.bin` and the command sequence in a text file and send it across? I can run it on my system (windows/Ubuntu) and confirm the error. If possible, can also get in a google meet to see and debug the issue on your system

**Comment by epic-coder-64** [comment]
You appear to be entering the commands correctly, so it is probably a boot sector issue. Can you please provide source to it?
